compress_jointcal_cfht_test_data.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
```

Comment:
These aren't installed files (not ending up in a `bin` directory) so there is no scheme in place for rewriting the shebang, but in this case it doesn't matter because the scripts don't load any LSST code.